### ISSUE TITLE
Issue with "<summary>": Does it really have an implicit role of button?

### ISSUE THREAD

**Issue by alattalatta** [mdn/content#13763]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary

#### What information was incorrect, unhelpful, or incomplete?

> Warning: Because the `<summary>` element has a default role of [button](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) (which strips all roles from child elements)...

#### Specific section or headline?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary#summaries_as_headings

#### What did you expect to see?

In [HTML Accessibility API Mappings 1.0](https://w3c.github.io/html-aam/#el-summary), there's no default role for `<summary>`, i.e. "No corresponding role". So, no such note at all.

"Implicit ARIA role" should be changed as well.

#### Did you test this? If so, how?

In both Chrome and Firefox's a11y tree, the heading's `heading` role is preserved.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/html/element/summary`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/summary/index.md
* Last commit: https://github.com/mdn/content/commit/303a6c3b8ec6658d2956c191c50aee0a0e9e4da5
* Document last modified: 2022-02-18T09:30:25.000Z

</details>

**Comment by sideshowbarker** [comment]
I think the https://w3c.github.io/html-aria/ spec is what normatively defines the implicit semantics for HTML elements. Anyway, at https://w3c.github.io/html-aria/#el-summary that spec at least does give `role=button` as the implicit semantics for `summary`.

Not sure what the AAM and ARIA in HTML specs are out of sync on this. @scottaohara and @stevefaulkner could speak to that.

**Comment by scottaohara** [comment]
it's "mostly" `button` at the platform api level.  it ranges from being exposed as a "button", "disclosure triangle", "summary" and no role depending on the browser/AT used. 

Honestly, it's messy and it needs to be fixed.  Here are just three issues about it (there are more):

https://github.com/w3c/html-aria/issues/355
https://github.com/whatwg/html/issues/2272
https://github.com/w3c/html-aam/issues/345

was having a bit of discussion about this in the aria wg for awhile.  need to pick it back up again sometime... but need to hammer down some proposals of what to do, cause something will have to change.

**Comment by estelle** [comment]
https://w3c.github.io/html-aria/#el-summary specifies that the implicit role for summary is button.

**Comment by estelle** [comment]
i assume at some point the role will switch to a new "summary" role, but until that point, we should not update the content. Closing.
